package.json

```diff
@@ -74,13 +72,14 @@
     "less": "^2.5.0",
     "less-loader": "^2.2.0",
     "mocha": "^2.2.4",
+    "progress-bar-webpack-plugin": "^1.0.1",
     "publish-release": "^1.1.0",
     "react-tools": "^0.13.1",
+    "redux-devtools": "^2.1.2",
     "request": "^2.65.0",
-    "redux-devtools"    : "^2.1.2",
-    "script-loader"     : "^0.6.1",
     "rewire": "^2.3.1",
     "rewire-webpack": "^1.0.0",
+    "script-loader": "^0.6.1",
     "sinon": "^1.14.1",
     "sinon-chai": "^2.7.0",
     "style-loader": "^0.12.3",
```

Comment:
Actually, is a fair question. But in that case, the text was reformatted 
after running 'npm install --save-dev'.

On 5/16/16 2:05 PM, loitly wrote:
>
> In package.json 
> <https://github.com/Caltech-IPAC/firefly/pull/85#discussion_r63425509>:
>
> >      "rewire": "^2.3.1",
> >      "rewire-webpack": "^1.0.0",
> > +    "script-loader": "^0.6.1",
> >      "sinon": "^1.14.1",
> >      "sinon-chai": "^2.7.0",
> >      "style-loader": "^0.12.3",
>
> not a criticism, just wanted to bring it up for discussion. when is it 
> appropriate to run 'reformat code'?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub 
> <https://github.com/Caltech-IPAC/firefly/pull/85/files/8bdfd737fd8631b45d10aff1ebb2742bee675e9e#r63425509>
>

-- 
Emmanuel Joliet
NASA/IPAC  Infrared Science Archive
MS 100-22, Caltech, Pasadena, CA 91125
Office: 170
Email:  ejoliet@ipac.caltech.edu
Phone:  (626) 395-1489